contracts/base/Settings.sol

```diff
@@ -409,6 +412,25 @@ contract Settings is Pausable, SettingsInterface {
         return isPauser(account);
     }
 
+    /**
+        @notice Sets a new escrow factory contract.
+        @param escrowFactoryAddress contract address of new escrow factory.
+     */
+    function setEscrowFactory(address escrowFactoryAddress) external onlyPauser() whenNotPaused() {
```

Comment:
I would validate the newEscrowFactoryAddress  is a contract / non empty address (and not equal to the current one).

I would remove the whenNotPaused() modifier. Just in case we need to update the reference.